vm/handler.go

```diff
@@ -60,6 +61,9 @@ func (h *DeployTxHandler) ProcessTx(
 	if tx.Value == nil {
 		value = loom.NewBigUIntFromInt(0)
 	} else {
+		if !common.IsPositive(tx.Value.Value) && !common.IsZero(tx.Value.Value) {
```

Comment:
If we are going to check the negative value, a feature flag is needed to prevent consensus failure